core-rust/state-manager/src/metrics.rs

```diff
@@ -134,62 +145,118 @@ impl CommittedTransactionsMetrics {
                     "committed_transactions_size",
                     "Size in bytes of committed transactions.",
                 ),
-                Self::buckets(DEFAULT_MAX_TRANSACTION_SIZE),
+                higher_resolution_for_lower_values_buckets_for_limit(DEFAULT_MAX_TRANSACTION_SIZE),
             )
             .registered_at(registry),
             execution_cost_units_consumed: new_histogram(
                 opts(
                     "committed_transactions_execution_cost_units_consumed",
                     "Execution cost units consumed per committed transactions.",
                 ),
-                Self::buckets(execution_config.cost_unit_limit as usize),
+                higher_resolution_for_lower_values_buckets_for_limit(
+                    execution_config.cost_unit_limit as usize,
+                ),
             )
             .registered_at(registry),
             substate_read_size: new_histogram(
                 opts(
                     "committed_transactions_substate_read_size",
                     "Total (per committed transaction) substate read size in bytes.",
                 ),
-                // TODO(RCnet-V3): update once max substate reads can be limited at execution
-                Self::buckets(DEFAULT_MAX_TOTAL_VERTEX_SUBSTATE_READ_SIZE),
+                // TODO: update once max substate reads can be limited at execution
```

Comment:
Why have you removed the RCnet-V3 from these comments?

Reply:
In general I'd like the TODOs to capture when we believe they need to be done for

Reply:
Not sure, probably this was `git merge`. I'll change back and merge.